src/main/java/com/google/sps/data/Room.java

```diff
@@ -153,4 +153,18 @@ private Room(List<Member> members, Queue<Video> videos, Queue<Message> messages,
     private List<EmbeddedEntity> getVideosAsEntities() {
         return this.videos.stream().map(Video::toEmbeddedEntity).collect(Collectors.toList());
     }
+
+    //Updates the current video in the room's state
+    public void updateCurrentVideoState(Video.VideoState state, long videoTimeStamp){
+        this.videos.peek().updateVideoState(state, videoTimeStamp);
```

Comment:
nit: this. getCurrentVideo(). updateVideoState

Reply:
This wasn't my code, sorry for adding it in the PR